pkg/util/nsenter/nsenter.go

```diff
@@ -83,20 +83,23 @@ func NewNsenter() *Nsenter {
 	}
 	// search for the required commands in other locations besides /usr/bin
 	for binary := range ne.paths {
-		// default to root
-		ne.paths[binary] = filepath.Join("/", binary)
+		found := false
 		for _, path := range []string{"/bin", "/usr/sbin", "/usr/bin"} {
 			binPath := filepath.Join(path, binary)
 			if _, err := os.Stat(filepath.Join(hostRootFsPath, binPath)); err != nil {
 				continue
 			}
 			ne.paths[binary] = binPath
+			found = true
 			break
 		}
-		// TODO: error, so that the kubelet can stop if the paths don't exist
-		// (don't forget that systemd-run is optional)
+
+		// systemd-run is optional
+		if binary != "systemd-run" && !found {
+			return nil, fmt.Errorf("unable to find binary %s for nsenter in any of /bin, /usr/sbin, or /usr/bin", binary)
```

Comment:
ideally make the paths a variable instead of listing it explicitly in 2 places (use strings.Join() here)